quickstart/src/browser.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+/**
+ * Add URL parameters to the web app URL.
+ * @param params - the parameters to add
+ */
+function addUrlParams(params) {
+  const combinedParams = Object.assign(getUrlParams(), params);
+  const serializedParams = Object.entries(combinedParams)
+    .map(([name, value]) => `${name}=${encodeURIComponent(value)}`)
+    .join('&');
+  history.pushState(null, '', `${location.pathname}?${serializedParams}`);
+}
+
+/**
+ * Generate an object map of URL parameters.
+ * @returns {*}
+ */
+function getUrlParams() {
+  const serializedParams = location.search.split('?')[1];
+  const nvpairs = serializedParams ? serializedParams.split('&') : [];
+  return nvpairs.reduce((params, nvpair) => {
+    const [name, value] = nvpair.split('=');
+    params[name] = decodeURIComponent(value);
+    return params;
+  }, {});
+}
+
+/**
+ * Whether the web app is running on a mobile browser.
+ * @type {boolean}
+ */
+const isMobile = (() => {
+  if (typeof navigator === 'undefined' || typeof navigator.userAgent !== 'string') {
+    return false;
+  }
+  return /Mobile/.test(navigator.userAgent);
```

Comment:
@manjeshbhargav Do we want to include tablets too? If so, we may need to test for the term `Tablet` as well.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent/Firefox#Mobile_and_Tablet_indicators

Reply:
@timmydoza I'm not depending on the word "Mobile" in the place where you are pointing to. It's at a different place in the user agent.

Reply:
My question is: do we want to unpublish local video tracks when a Firefox tablet user backgrounds the app? If so, then this should be:
`  return /Mobile|Tablet/.test(navigator.userAgent);`